### PR TITLE
update some text

### PR DESCRIPTION
**Pull request by ryanatkn**

### CODE REVIEW

File: src/routes/about/+page.svelte [hunk 1]
```diff
@@ -85,28 +85,34 @@
...
-				We expect our tech — which is focused on <em>communities</em>, not <em>messaging</em> — to eventually
-				support a decentralized protocol standard, perhaps with ActivityPub or Matrix or both, but today
-				we are focused on UX quality and implementation velocity. We find it much easier to deliver good
-				UX with a centralized architecture given the nature of decentralized systems and our small self-funded
-				team with no expertise in such things. If Matrix fits your usecases, we recommend considering
-				using it instead of Felt. For microblogging, we recommend Mastodon.
+				We expect our tech — which is focused on <em>communities</em>, not <em>messaging</em> — to
+				eventually support a decentralized protocol standard, perhaps with ActivityPub or Matrix or
+				both, but today we are focused on UX quality and implementation velocity. It's much easier
+				for our small self-funded team to deliver good UX with a centralized architecture given the
```
Comment (greatbacon) [inline src/routes/about/+page.svelte:98]:
```suggestion
				for our small, self-funded team to deliver good UX with a centralized architecture given the
```

File: src/routes/about/+page.svelte [hunk 2]
```diff
@@ -85,28 +85,34 @@
...
 			<p>
 				Felt's focus is on communities and their custom needs, with an emphasis on delivering a
 				polished UX in smaller scale, higher trust environments. We are however attempting to follow
-				the standard ActivityPub/Fediverse/Mastodon/etc
+				the standard ActivityPub/Fediverse/Mastodon
 				<a href="https://www.w3.org/TR/activitystreams-vocabulary/">ActivityStreams vocabulary</a>,
-				(but not ActivityPub the federated protocol, not yet) setting us up for potential future
-				compatibility. We're also maintaining JSON schemas that can be used in any programming
-				language, so at least our off-spec stuff is machine-readable.
+				(but not ActivityPub the federated protocol, not yet), setting us up for potential future
```
Comment (greatbacon) [inline src/routes/about/+page.svelte:111]:
```suggestion
				setting us up for potential future
```
do we need this parenthetical?

File: src/routes/about/+page.svelte [hunk 3]
```diff
@@ -85,28 +85,34 @@
...
+				of Felt. For microblogging and other
+				<a href="https://publicinfrastructure.org/2023/03/23/new_public-a-social-network-taxonomy/"
+					>big room social media</a
+				>, we recommend Mastodon and the Fediverse.
 			</p>
 			<p>
 				Felt's focus is on communities and their custom needs, with an emphasis on delivering a
 				polished UX in smaller scale, higher trust environments. We are however attempting to follow
-				the standard ActivityPub/Fediverse/Mastodon/etc
+				the standard ActivityPub/Fediverse/Mastodon
```
Comment (greatbacon) [inline src/routes/about/+page.svelte:109]:
```suggestion
				the standard
```
This seems kind of verbose

File: src/routes/about/+page.svelte [hunk 4]
```diff
@@ -85,28 +85,34 @@
...
 				<a href="https://www.w3.org/TR/activitystreams-vocabulary/">ActivityStreams vocabulary</a>,
-				(but not ActivityPub the federated protocol, not yet) setting us up for potential future
-				compatibility. We're also maintaining JSON schemas that can be used in any programming
-				language, so at least our off-spec stuff is machine-readable.
+				(but not ActivityPub the federated protocol, not yet), setting us up for potential future
+				compatibility. We also maintain
+				<a href="https://github.com/feltjs/felt-server/blob/main/src/static/schemas/vocab.json"
+					>JSON schemas</a
+				> that can be used in any programming language, so at least our off-spec stuff is machine-readable.
```
Comment (greatbacon) [inline src/routes/about/+page.svelte:115]:
```suggestion
				> that can be used in any programming language.
```
I don't know if we need this hedge here either

**Review by greatbacon — APPROVED** [review]
Left a couple of notes, but feel free to ignore them!

**Comment by ryanatkn** [comment]
good edits, ty, accepted them all

**Pull request merged**